### ISSUE TITLE
Logs show "Executed action (null)" for razor pages

### ISSUE THREAD

**Issue by davidfowl** [aspnet/Mvc#6390]
```
info: Microsoft.AspNetCore.Mvc.RazorPages.Internal.PageActionInvoker[2]
      Executed action (null) in 727.489ms
```

The logs probably need to be updated to handle this and show the page name being executed instead.

/cc @rynowak @DamianEdwards

**Comment by rynowak** [comment]
This was fixed in b796bc0f39d08bdbf0ce36279db3148ed9c9e888 in 2.0.0-preview3

**Comment by rynowak** [comment]
Fix is here: https://github.com/aspnet/Mvc/commit/b796bc0f39d08bdbf0ce36279db3148ed9c9e888#diff-851c2798f118fd53e78499834de23bf4R41
